docs/source/cli/model-training-inference/distributed/sagemaker.rst

```diff
@@ -21,73 +21,20 @@ GraphStorm uses SageMaker's **BYOC** (Bring Your Own Container) mode. Therefore,
 
 .. _build_sagemaker_docker:
 
-Step 1: Build a SageMaker-compatible Docker image
-...................................................
+Building and pushing a SageMaker uses the same scripts as for building a local image,
+described in :ref:`GraphStorm Docker build instructions <_build_docker>`.
 
-.. note::
-    * Please make sure your account has access key (AK) and security access key (SK) configured to authenticate accesses to AWS services.
-    * For more details of Amazon ECR operation via CLI, users can refer to the `Using Amazon ECR with the AWS CLI document <https://docs.aws.amazon.com/AmazonECR/latest/userguide/getting-started-cli.html>`_.
-
-First, in a Linux machine, configure a Docker environment by following the `Docker documentation <https://docs.docker.com/get-docker/>`_ suggestions.
-
-In order to use the SageMaker base Docker image, users need to use the following command to authenticate to pull SageMaker images.
+In short you can run the following:
 
 .. code-block:: bash
 
-    aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin 763104351884.dkr.ecr.us-east-1.amazonaws.com
-
-Then, clone GraphStorm source code, and build a GraphStorm SageMaker compatible Docker image from source with commands:
-
-.. code-block:: bash
-
-    git clone https://github.com/awslabs/graphstorm.git
-
-    cd /path-to-graphstorm/docker/
-
-    bash /path-to-graphstorm/docker/build_docker_sagemaker.sh /path-to-graphstorm/ <DEVICE_TYPE> <IMAGE_NAME> <IMAGE_TAG>
-
-The ``build_docker_sagemaker.sh`` script takes four arguments:
-
-1. **path-to-graphstorm** (**required**), is the absolute path of the ``graphstorm`` folder, where you cloned the GraphStorm source code. For example, the path could be ``/code/graphstorm``.
-2. **DEVICE_TYPE** (optional), is the intended device type of the to-be built Docker image. There are two options: ``cpu`` for building CPU-compatible images, and ``gpu`` for building Nvidia GPU-compatible images. Default is ``gpu``.
-3. **IMAGE_NAME** (optional), is the assigned name of the to-be built Docker image. Default is ``graphstorm``.
+    cd graphstorm/
+    bash docker/build_graphstorm_image.sh --environment sagemaker
+    bash docker/push_graphstorm_image.sh --environment sagemaker --region "us-east-1" --account "123456789012"
+    # Will push an image to '123456789012.dkr.ecr.us-east-1.amazonaws.com/graphstorm:sagemaker-gpu'
```

Comment:
Shall we add any additional comment for error for AWS credential?

Reply:
We may need to add some docs about AWS credential

Reply:
Added